crates/bevy_math/src/bounding/raycast2d.rs

```diff
@@ -224,21 +229,29 @@ mod tests {
                 4.996,
             ),
         ] {
-            let case = format!(
+            assert!(
+                test.intersects(volume),
                 "Case:\n  Test: {:?}\n  Volume: {:?}\n  Expected distance: {:?}",
-                test, volume, expected_distance
+                test,
+                volume,
+                expected_distance
```

Comment:
The variables for all the assert messages could be inside the string, like this:
```rs
"Case:\n Test: {test:?}\n Volume: {volume:?}\n Expected distance: {expected_distance:?}"
```

Reply:
Yeah that makes sense, shaves a few lines off the diff too!